src/main/java/net/minecraftforge/client/ClientForgeMod.java

```diff
@@ -36,7 +36,6 @@ public static void onRegisterGeometryLoaders(ModelEvent.RegisterGeometryLoaders
 
         // TODO: Deprecated. To be removed in 1.20
         event.register("bucket", DynamicFluidContainerModel.Loader.INSTANCE_DEPRECATED);
```

Comment:
Why are you only removing 1 of the three loaders?

Reply:
The others are unrelated to this change.